backend/tasks/inference_tasks.py

```
@@ -0,0 +1,458 @@
+        input_text: The context text
+        input_questions: List of questions
+        input_options: List of option lists (one per question)
+    
+    Returns:
+        List of predicted answers
+    """
+    try:
+        logger.info(f"Starting MCQ answer prediction. Questions: {len(input_questions)}")
+        
+        outputs = []
+        for question, options in zip(input_questions, input_options):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)